src/hotspot/share/c1/c1_Runtime1.hpp

```diff
@@ -135,6 +135,8 @@ class Runtime1: public AllStatic {
   static void initialize(BufferBlob* blob);
   static void initialize_pd();
 
+  static uint runtime_blob_current_thread_offset(frame f);
```

Comment:
I think this returns an offset in wordSize units, but it's not documented.  In some places we always return an offset in bytes and let the caller convert.

Reply:
Added comment.